CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+# Version 3.3.0 GA
+* Update OpenTelemetry to 1.15 [#2332](https://github.com/microsoft/ApplicationInsights-Java/pull/2332).
+* Update dependencies [#2333](https://github.com/microsoft/ApplicationInsights-Java/pull/2333).
+* Simplify GitHub actions [#2340](https://github.com/microsoft/ApplicationInsights-Java/pull/2340).
+* Add configuration for disk persistence capacity [#2329](https://github.com/microsoft/ApplicationInsights-Java/pull/2329).
+* Change ai.cloud.role format [#2326](https://github.com/microsoft/ApplicationInsights-Java/pull/2326).
+* Add support for runtime attach [#2325](https://github.com/microsoft/ApplicationInsights-Java/pull/2325).
+* Minor Statsbeat improvements [#2322](https://github.com/microsoft/ApplicationInsights-Java/pull/2322).
```

Comment:
since not a user facing change
```suggestion
* Minor Statsbeat improvements [#2322](https://github.com/microsoft/ApplicationInsights-Java/pull/2322).
```